doc/rest.md

```diff
@@ -158,7 +158,7 @@ JSON array of objects representing posted layers with following structure:
    - **layman_original_parameter**: name of the request parameter that contained the file name; currently, the only possible value is `file`
 
 ### DELETE Workspace Layers
-Delete existing layers and all associated sources, including vector data file and DB table for all layers in the workspace. It is possible to delete layers, whose publication process is still running. In such case, the publication process is aborted safely. Only layers on which user has [write access right](./security.md#access-to-multi-publication-endpoints) are deleted.
+Delete existing layers and all associated sources, including vector data file and DB table for all layers in the workspace. [Asynchronous tasks](async-tasks.md) of affected layers are aborted. Only layers on which user has [write access right](./security.md#access-to-multi-publication-endpoints) are deleted.
```

Comment:
'[Asynchronous tasks](async-tasks.md) of affected layers are aborted.' -> 'Currently running [asynchronous tasks](async-tasks.md) of affected layers are aborted.'